app/controllers/employees_controller.rb

```diff
@@ -12,8 +12,10 @@ def new
   def edit; end
 
   def create
-    @employee = Employee.new(employee_params)
+    salary_params = employee_params.dig(:salaries_attributes, "0")
```

Comment:
I don't love this but I was having trouble making the employee model validate on create.

Reply:
I wonder if this validation could be handled in a `before_action` method?

Reply:
Could adding `:inverse_of` to the `salary` model fix this?

https://github.com/rails/rails/issues/8828